docs/help/security.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Security
+sidebar_position: 2.8
+description:
+  "Overview of security measures and practices within Hasura DDN, including data encryption, access control, and
+  compliance with industry standards."
+keywords:
+  - hasura ddn
+  - security
+  - data encryption
+  - access control
+  - compliance
+  - metadata encryption
+  - data transmission
+  - authentication
+  - multi-factor authentication
+  - MFA
+  - SAML
+  - SSO
+---
+
+# Security in Hasura DDN
+
+## Introduction
+
+Hasura DDN is designed with a strong emphasis on security, ensuring that your data and APIs are protected through
+various measures. This document outlines the key security features and practices implemented within Hasura DDN, covering
+data encryption, access control, network security, and authentication.
+
+## Data Encryption
+
+### Encryption at Rest
+
+All metadata stored within Hasura DDN is encrypted at rest using the **AES-GCM-256** algorithm. This industry-standard
+encryption ensures that your data is secure even if the underlying storage is compromised. Hasura also uses **Envelope
+encryption** for the keys themselves.
+
+### Encryption in Transit
+
+All communication between your client applications, the Hasura DDN engine, and configured data connectors is secured
+using **HTTPS/TLS 1.2** or higher. This ensures that your data is encrypted during transmission, protecting it from
+interception or tampering.
+
+### Network Connectivity Requirements
+
+When using Hasura public DDN connectors to connect to your data sources, these sources must be accessible from the
+internet (allowing connections from `0.0.0.0/0`). This is because Hasura DDN operates from dynamic IP addresses and does
+not provide static IP addresses for connections.
+
+If your security requirements necessitate the use of static IP addresses or private network connectivity, consider using
+[Hasura Private DDN](/private-ddn/overview.mdx), which allows you to host the data plane within your own infrastructure.
+
+## Metadata Storage and Handling
+
+### Types of Metadata Stored
+
+Hasura DDN's control plane stores metadata related to your project's configuration. This metadata includes:
+
+- **HML (Hasura Metadata Language) files:** These files define the structure of your API, including data source
+  information such as table names, column names, relationships, and permissions.
+- **Subgraph information:** Details about your project's subgraphs.
+- **User access information:** Data on which users have access to the supergraph and subgraphs.
```

Comment:
"Data on which users have access..." can be reworded. Use of word "Data" is confusing; at a quick glance it feels this refers to actual data being stored.
suggestion: "RBAC configurations that determines supergraph and subgraphs access for users"

Reply:
👍